skycoin/dmsg
# Fix/add redis retry

Fixes #190 

 Changes:	
- Added retry logic to redis in dmsg discovery
- Fixes appveyor windows build tests

How to test this PR:
1. Build `make build`
2. Start dmsg discovery with `./bin/dmsg-discovery` (make sure redis is not working)
3. Check retry logic
